### ISSUE TITLE
Date entry broken on CM 12.1 (Android 5.1.1)

### ISSUE THREAD

**Issue by hoyalu** [koelleChristian/trickytripper#11]
In the date entry dialog of Tricky Tripper 1.5.5, I can only change the day of the month, the year (only one by one, no scrolling after tapping the year), and the time, but not the month (impossible to scroll). Also, the dialog looks a little ugly. Turning to landscape doesn't improve things. I have no idea if this bug is related to bug #5 which refers to Android 6.0.
![screenshot_2016-06-01-23-16-07](https://cloud.githubusercontent.com/assets/19696994/15726230/345d5e02-283f-11e6-8d36-fd759f84048b.png)

**Comment by koelleChristian** [comment]
Thanks for your feedback. I will check this out on the weekend.

I had fixed something in 1.5.4 where by making the date-time-chooser scrollable.

Sorry I have no time before that.

Regards and thanks

**Comment by koelleChristian** [comment]
TT 1.5.5 date & time picker works nice on my Fairphone 2 (Android 5.1 based) and the Android 6 and 5.1 Emulator.

The solution for #5 was to put the entire datepicker popup into a scrolling frame, so TT 1.5.5 should allow you to scroll the entire content of the screenshot you provided.

This issue looks the same as #5 to me.  I lack the hardware to reproduce it.

One suggestion might be to uninstall the app an reinstall it. Before you do that, I suggest you to export your trips. A common backup tool should allow you to backup the app data and should allow you to bring it back. I used Helium for such a purpose in the past.

**Comment by koelleChristian** [comment]
I am prepared to close this issue as I am unable to reproduce it with anything available to me.
Have you tried as suggested?

**Comment by hoyalu** [comment]
I've tried as suggested (with Oandbackup). No success. Getting rid of the line break by changing the language or the screen resolution was another try, but no success here either.

On CM 13.0 (Android 6.0.1) the date picker works fine for me. I get arrows pointing to the left and to the right that let me change the month.

Unfortunately, I will have to stick to CM 12.1 as long as offline calendars don't work with 13.0. So for me the bug remains, but I can see the light at the end of the tunnel. Maybe this bug should be closed. Maybe not, because others may have to stick to Android 5.1 as well.

Thank you for your efforts!

**Comment by koelleChristian** [comment]
You are right. I will leave this open and pray that CM 13.0 will be available for further devices, including my Sony scorpion_windy, where I would prefer a CM over the Android 6 it comes with.
